loader/CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ if(WIN32)
 
     # Configure the file to include the versioning info
     # Place it in the current directory for check-in - so the GN build has up to date info
-    configure_file(loader.rc.in ${CMAKE_CURRENT_BINARY_DIR}/loader.rc)
+    configure_file(loader.rc.in ${CMAKE_CURRENT_SOURCE_DIR}/loader.rc)
```

Comment:
Does this mean building with CMake can dirty the source tree? I think the CMake build should continue to use `CMAKE_CURRENT_BINARY_DIR` if it is going to generate its own rc file.

Reply:
Yes? The problem with what you say is making sure to use the 'right' loader.rc file when building. 
The GN build needs to have it checked in, but we also need to be able to set the version using a command line configure argument. Its possible to have the CMake originating file be put in the BINARY dir, then use that, but does add some complexity.

Reply:
Yes you would end up with 2 rc files, but CMake should never know about the checked-in one--it is merely provided for compatibility with non-CMake build systems.

Reply:
I think the issue is resolved, as now CMake will only configure loader.rc if BUILD_DLL_VERSIONINFO was set, but it will place it in the build directory. Then it will only use the file in the build folder if BUILD_DLL_VERSIONINFO was set, and will use the checked in file otherwise.